ingestors/npm.go

```diff
@@ -104,15 +99,15 @@ func (ingestor *NPM) Ingest(results chan data.PackageVersion) {
 }
 
 func (ingestor *NPM) SetLatestSequence(seq string) error {
-	err := ingestor.redisClient.Set(context.Background(), NPMLatestSequenceRedisKey, seq, 0).Err()
+	err := redis.Client.Set(context.Background(), NPMLatestSequenceRedisKey, seq, 0).Err()
```

Comment:
Any thoughts to just exporting a Set/Get/SetNX and keeping "Client" internal?

Reply:
Could def do that, but then we'll have to whack-a-mole every function we use. I'm fine either way, WDYT?

Reply:
It's probably fine, I don't have a particularly compelling reason to change it other than it's slightly cleaner looking.